libriscan/biblios/templates/biblios/components/layout/welcome_section.html

```diff
@@ -74,7 +74,251 @@ <h2 class="text-4xl font-bold mb-3">Welcome to LibriScan!</h2>
       </div>
     </div>
 
+    <!-- Tabs with Icons -->
+    <div class="tabs tabs-boxed tabs-lg mb-6" id="main-tabs">
+      <a class="tab tab-active gap-2" data-tab="tab1">
```

Comment:
Per comment on removing `homepage.css` also use meaningful naming instead of tab1
```suggestion
              <a class="tab tab-active gap-2 transition-all duration-300 hover:bg-base-300 hover:-translate-y-0.5" data-tab="tab1">
```